bench/bench.js

```diff
@@ -1,4 +1,4 @@
-import {earcut, flatten} from '../src/earcut.js';
+import earcut, {flatten} from '../src/earcut.js';
```

Comment:
I had to make this tweak to the benchmarks to get them to run

main:

```
typical OSM building (15 vertices): x 2,751,261 ops/sec ±0.50% (96 runs sampled)
dude shape (94 vertices): x 76,587 ops/sec ±0.42% (100 runs sampled)
dude shape with holes (104 vertices): x 61,820 ops/sec ±0.32% (100 runs sampled)
complex OSM water (2523 vertices): x 1,024 ops/sec ±0.49% (97 runs sampled)
```

this branch:

```
typical OSM building (15 vertices): x 2,644,854 ops/sec ±0.46% (92 runs sampled)
dude shape (94 vertices): x 76,872 ops/sec ±0.31% (100 runs sampled)
dude shape with holes (104 vertices): x 61,708 ops/sec ±0.28% (100 runs sampled)
complex OSM water (2523 vertices): x 1,040 ops/sec ±0.22% (98 runs sampled)
```